send/config.c

```diff
@@ -80,6 +80,44 @@ static int smtp_auth_validator(const struct ConfigSet *cs, const struct ConfigDe
   return CSR_SUCCESS;
 }
 
+/**
+ * sendmail_validator - Validate the "sendmail" config * variable - Implements ConfigDef::validator() - @ingroup cfg_def_validator
+ */
+static int sendmail_validator(const struct ConfigSet *cs, const struct ConfigDef *cdef,
```

Comment:
We can rename this to, er... `command_validator`, maybe (see below)
(and adjust the comment)